python_modules/dagster/dagster/core/definitions/version_strategy.py

```diff
@@ -93,7 +75,11 @@ def _get_source_hash(self, fn):
         return hashlib.sha1(code_as_str.encode("utf-8")).hexdigest()
 
     def get_solid_version(self, context: SolidVersionContext) -> str:
-        return self._get_source_hash(context.solid_def.compute_fn.decorated_fn)
+        compute_fn = context.solid_def.compute_fn
```

Comment:
I think it's worth switching this to `get_op_version`, because users might look to it as an example of how to implement a `VersionStrategy`.